src/main/java/teammates/ui/controller/InstructorFeedbackQuestionCopyPageData.java

```diff
@@ -0,0 +1,48 @@
+package teammates.ui.controller;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import teammates.common.datatransfer.AccountAttributes;
+import teammates.common.datatransfer.FeedbackQuestionAttributes;
+import teammates.common.datatransfer.FeedbackQuestionDetails;
+import teammates.ui.template.FeedbackQuestionCopyTable;
+import teammates.ui.template.FeedbackQuestionTableRow;
+
+public class InstructorFeedbackQuestionCopyPageData extends PageData {
+
+    private final List<FeedbackQuestionAttributes> questions;
+    
+    public InstructorFeedbackQuestionCopyPageData(AccountAttributes account,
+                                                  List<FeedbackQuestionAttributes> copiableQuestions) {
```

Comment:
Perhaps this could be written with a higher level line break:
```
public InstructorFeedbackQuestionCopyPageData(
        AccountAttributes account, List<FeedbackQuestionAttributes> copiableQuestions) {
```